integration/combination/test_function_with_alias.py

```diff
@@ -35,6 +35,42 @@ def test_updating_version_by_changing_property_value(self):
         self.assertEqual(len(alias), 1)
```

Comment:
Should also include transform test (you can use the existing integration test template) to ensure the the hash doesn't change unexpectedly.